README.md

```diff
@@ -50,6 +50,23 @@ But also in our hall of fame:
 
 Feel free to ask questions, enhancements and to contribute to this project!
 
+## Development
+
+The project is managed with `hatch`. [Install `hatch`](https://hatch.pypa.io/1.9/install/#gui-installer) first.
+
+Run the tests:
+
+```
+hatch run test:test
+hatch run style:check
+```
+
+Serve the documentation:
+
+```
+hatch run doc:serve
+```
+
 ## Credits
 
-- Logo by [max.icons](https://www.flaticon.com/authors/maxicons)
\ No newline at end of file
+- Logo by [max.icons](https://www.flaticon.com/authors/maxicons)
```

Comment:
what did you chance on this line, maybe you added a new line?

please try making sure it does not show in the diff

Reply:
I think, this might be inessential to the diff.
My editor probably added a line at the end.

Reply:
Can you remove it then? this change should not be present in the diff